mscore/easeinoutcanvas.cpp

```diff
@@ -0,0 +1,192 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2010-2019 Werner Schweer and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#include "easeinoutcanvas.h"
+#include "..\libmscore\easeInOut.h"
```

Comment:
I think you can and should drop that ` ..\`, maybe even `..\libmscore\`. And should change \ to /

Reply:
Guess that is the reason why it fails to build on Mac and Linux

Reply:
Done

Reply:
Looks much better now ;-)